packages/core/src/controllers/LionWebBulkApi.ts

```diff
@@ -20,7 +21,9 @@ class LionWebBulkApiImpl implements LionWebBulkApi {
      * @param res The list of all partition nodes, without children or annotations
      */
     async partitions(req: Request, res: Response): Promise<void> {
-        // const result = [];
+        if (requestsVerbosity) {
+            console.log(` * partitions request received, with body of ${req.headers["content-length"]} bytes`)
```

Comment:
Is it an idea to do all logging using an object with methods `requestLog`, `dbError`, etc. That object (and its methods) are aware of the value of `requestsVerbosity` etc. so these don't become separate global objects.

Reply:
I like the idea of a separate Logging object so we don't need the `if` around every log statement.

Reply:
I introduce the Logger object, as suggested